dvc/commands/move.py

```diff
@@ -40,3 +40,17 @@ def add_parser(subparsers, parent_parser):
     ).complete = completion.FILE
     move_parser.add_argument("dst", help="Destination path.").complete = completion.FILE
     move_parser.set_defaults(func=CmdMove)
+
+    mv_parser = subparsers.add_parser(
+        "mv",
+        parents=[parent_parser],
+        description=append_doc_link(MOVE_DESCRIPTION, "move"),
+        help=MOVE_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+
+    mv_parser.add_argument(
+        "src", help="Source path to a data file or directory."
+    ).complete = completion.FILE
+    mv_parser.add_argument("dst", help="Destination path.").complete = completion.FILE
+    mv_parser.set_defaults(func=CmdMove)
```

Comment:
I think you should be able to just use:

```
aliases=["mv"],
```

like in https://github.com/iterative/dvc/blob/f47d21b0c29f5603bff09cc51c4a14a1bdf962c6/dvc/commands/ls/__init__.py#L74C9-L74C24

Reply:
Oh cool, didn't know that!